grpcsupport/grpc_client.py

```diff
@@ -29,6 +33,17 @@ def __init__(self, name, team, index):
                 print("Exception when trying to connect to grpc server: " + str(e))
                 pass
 
+        def get_extra_pids(self):
```

Comment:
I tinkered with this a little, but the multiprocessing situation is very complicated.

However, blocking is probably fine: If an agent is taking a long time to get process ids, that's probably because it hasn't started up yet, so it's screwed for the first kickoff anyhow. It won't block the other agent from running.

I'm inclined to leave this as-is.